### PR TITLE
Update our Windows CI leg to use the non-Hosted Windows queue

### PR DESCRIPTION
**Pull request by eerhardt**
We are getting a random test hang using the hosted Windows queue. Changing to a non-hosted queue so we will be able to debug it if it happens again.

/cc @shauheen

### CODE REVIEW

File: .vsts-dotnet-ci.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@ phases:
     name: Windows_NT
     buildScript: build.cmd
     queue:
-      name: Hosted VS2017
+      name: DotNetCore-Windows
```
Comment (Ivanidzo4ka) [inline .vsts-dotnet-ci.yml]: >DotNetCore-Windows [](start = 12, length = 18)

Could not find a pool with name DotNetCore-Windows. The pool does not exist or has not been authorized for use. For authorization details, refer to [https://aka.ms/yamlauthz](https://aka.ms/yamlauthz)

Comment (eerhardt) [reply .vsts-dotnet-ci.yml]: :(  @ilyas1974 - any ideas? Did I misspell it?

Comment (eerhardt) [reply .vsts-dotnet-ci.yml]: @dotnet/dnceng @chcosta - any ideas on the issue here? What queue name should I use?

Comment (safern) [reply .vsts-dotnet-ci.yml]: The issue here is that I think this pool doesn't exist in the public domain.

Comment (dagood) [reply .vsts-dotnet-ci.yml]: When I get that, I make some dummy change and re-save the build def (on the Azure DevOps side) to get it authorized. The pool shows up for me in the public project: https://dnceng.visualstudio.com/public/_settings/agentqueues?queueId=28&_a=agents. I think work is being done to improve this experience.

**Pull request merged**